docs/developer/getting-started.md

```diff
@@ -62,6 +62,8 @@ In the background, `npm start` makes a [concurrently](https://github.com/kimmobr
 
 Once the angular server starts, it takes some time to pre-compile all assets before serving them. By default, the angular development server watches for file changes and will update accordingly.
 
+If you want to test your local language in the web ui by i18n, you should run dashboard with the command: `npm run start:prod`.
```

Comment:
There is a section about building prod build a bit lower already. I'd only add a small note there regarding i18n.

Current:

> The code is compiled, compressed and debug support removed. The dashboard binary can be found in the `dist` folder.
To build and immediately serve Dashboard from the `dist` folder, use the following task:

Proposed change
> The code is compiled, compressed, i18n support is enabled and debug features are removed. The dashboard binary can be found in the `dist` folder.
To build and immediately serve Dashboard from the `dist` folder, use the following task:

Reply:
ok, I will add this change.

Reply:
done